deltachat-ios/Controller/QrViewController.swift

```diff
@@ -6,13 +6,7 @@ class QrViewController: UIViewController {
 
     private let dcContext: DcContext
 
-    private var contact: DcContact? {
```

Comment:
DcContact was only needed to create the hint message below the QR code, which is now injected in instanciation